.travis.yml

```diff
@@ -3,12 +3,20 @@ language: node_js
 node_js:
   - 'stable'
 
+before_install:
```

Comment:
FYI - I got this code from here: https://code.visualstudio.com/docs/extensions/testing-extensions#_running-tests-automatically-on-travis-ci-build-machines